vspace/src/main/java/edu/asu/diging/vspace/core/aspects/ExhibitionDataAspect.java

```diff
@@ -42,4 +50,22 @@ public void setExhibition(JoinPoint jp) {
             }
         }
     }
+    
+    @Around("execution(public * edu.asu.diging.vspace.web..ExhibitionSpaceController.*(..))")
+    public Object showExhibition(ProceedingJoinPoint jp) throws Throwable {
+        Object[] args = jp.getArgs();
+        Exhibition exhibition = (Exhibition) exhibitionManager.getStartExhibition();
+        AuthenticationFacade authFacade = new AuthenticationFacade();
+        if (exhibition.getMode().equals(ExhibitionModes.ACTIVE.getValue())) {
+            return jp.proceed();
+        }
+        else if(!(authFacade.getAuthenticatedUser() == null)){
+           ((Model) args[1]).addAttribute("showModal","true");
```

Comment:
the modal might not always be the second argument, this will throw an exception if it is not.